src/wp-includes/global-styles-and-settings.php

```diff
@@ -185,15 +185,11 @@ function wp_get_global_stylesheet( $types = array() ) {
 function wp_get_global_styles_svg_filters() {
 	// Return cached value if it can be used and exists.
 	// It's cached by theme to make sure that theme switching clears the cache.
-	$can_use_cached = (
-		( ! defined( 'WP_DEBUG' ) || ! WP_DEBUG ) &&
-		( ! defined( 'SCRIPT_DEBUG' ) || ! SCRIPT_DEBUG ) &&
-		( ! defined( 'REST_REQUEST' ) || ! REST_REQUEST ) &&
-		! is_admin()
-	);
-	$transient_name = 'global_styles_svg_filters_' . get_stylesheet();
+	$can_use_cached = ! WP_DEBUG && ! is_admin();
```

Comment:
Let's add the same comment added in the other usages of this approach (e.g. https://github.com/WordPress/wordpress-develop/commit/dd33e32767d119929e769cd21abc3b00690fb14f#diff-1be8a8089d87d1b4b6e449c0967680d6f7c170d13f247180a975cbea5e54070fR88) here too. Also, like in that linked code, the `! is_admin()` here is not necessary now and should be removed.
```suggestion
	/*
	 * Ignore cache when `WP_DEBUG` is enabled, so it doesn't interfere with the theme
	 * developer's workflow.
	 *
	 * @todo Replace `WP_DEBUG` once an "in development mode" check is available in Core.
	 */
	$can_use_cached = ! WP_DEBUG;
```

Reply:
The logic for `is_admin`, `script_debug`, and `rest_request` was removed from `wp_get_global_stylesheet` for the reasons I documented here https://github.com/WordPress/gutenberg/pull/45679/files#r1025439309 So I suppose we can also remove it from here as well. @ajlende would you remember any other case in which these would be necessary?

Reply:
Fixed in https://github.com/WordPress/wordpress-develop/pull/3926/commits/5448f9cbd2d07620829437346aae973de112dc33

Reply:
> @ajlende would you remember any other case in which these would be necessary?

Nope, the SVG stuff was supposed to work exactly the same as the stylesheets.